src/quo/components/wallet/amount_input/schema.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo.components.wallet.amount-input.schema)
+
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:status {:optional true} [:enum :default :error]]
+      [:theme :schema.common/theme]
+      [:on-change-text {:optional true} [:maybe fn?]]
+      [:container-style {:optional true} [:maybe :map]]
+      [:auto-focus {:optional true} [:maybe :boolean]]
+      [:min-value {:optional true} [:maybe :int]]
+      [:max-value {:optional true} [:maybe :int]]
+      [:return-key-type {:optional true} [:maybe :keyword]]
```

Comment:
if we know ahead of time the possible key-types, this could be also an enum.

Reply:
It will be the list of all of the TextInput's [returnKeyType param](https://reactnative.dev/docs/textinput#returnkeytype) values. Should I just list them all?
Also, it may vary from one RN version to another. Very rarely, but it is possible.

Reply:
i guess this should be done for the schema of the text-input rather than here